lib/maze/bitbar_utils.rb

```diff
@@ -47,6 +50,35 @@ def upload_app(api_key, app)
         end
         app_uuid
       end
+
+      # Starts the BitBar local tunnel
+      # @param bb_local [String] path to the SBSecureTunnel binary
+      # @param username [String] Username to start the tunnel with
+      # @param access_key [String] BitBar access key
+      def start_local_tunnel(bb_local, username, access_key)
+        $logger.info 'Starting BitBar SBSecureTunnel local tunnel'
+        command = "#{bb_local} --username #{username} --authkey #{access_key}" \
+                    "--ready #{BB_READY_FILE} --kill #{BB_KILL_FILE}"
+
+        @tunnel_shell = Maze::InteractiveCLI.new
+        @tunnel_shell.run_command(command)
+        success = Maze::Wait.new(timeout: 30).until do
```

Comment:
Is 30 seconds likely to be plenty?  Of course there's a balance, though - we don't want to wait too long if it's a genuine failure case.

Reply:
From what I can tell it starts pretty quickly, if it goes for 30 seconds without starting it's failed